browser/src/slideshow/LayerDrawing.ts

```diff
@@ -105,6 +105,8 @@ class LayerDrawing {
 	private onSlideRenderingCompleteCallback: VoidFunction = null;
 	private layerRenderer: LayerRenderer;
 	private videoRenderers: Map<string, Array<VideoRenderer>> = new Map();
+	private isTransitionActive: boolean = false;
+	private queuedLayers: LayerInfo[] = [];
```

Comment:
I see a type mismatch here; you use `queueLayers` for keeping track of events `e` which are not of type LayerInfo which is the type of `e.message`.